benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -313,7 +314,13 @@ private QueryExecutionResult executeSingleQuery(
     private Connection getConnectionFor(QueryExecution queryExecution)
             throws SQLException
     {
-        return applicationContext.getBean(queryExecution.getBenchmark().getDataSource(), DataSource.class).getConnection();
+        try {
+            return applicationContext.getBean(queryExecution.getBenchmark().getDataSource(), DataSource.class).getConnection();
+        }
+        catch (Exception exception) {
+            LOG.error("Getting connection failed: {}\n{}", exception.getMessage(), ExceptionUtils.stackTraceToString(exception));
```

Comment:
why throwing exception is not sufficient? it should contain full error message with stacktrace